internal/cli/cmd/connectivity.go

```diff
@@ -120,6 +121,9 @@ func newCmdConnectivityTest() *cobra.Command {
 	cmd.Flags().BoolVarP(&params.Debug, "debug", "d", false, "Show debug messages")
 	cmd.Flags().BoolVarP(&params.PauseOnFail, "pause-on-fail", "p", false, "Pause execution on test failure")
 	cmd.Flags().BoolVar(&params.SkipIPCacheCheck, "skip-ip-cache-check", true, "Skip IPCache check")
+	cmd.Flags().BoolVar(&params.Perf, "perf", false, "Run network Performance tests")
+	cmd.Flags().IntVar(&params.Duration, "duration", 30, "Duration for the Performance test to run, Default:30s")
```

Comment:
I'd suggest to rename this to `--perf-duration` so it's apparent from the name that this relates to performance tests:

```suggestion
	cmd.Flags().IntVar(&params.Duration, "perf-duration", 30, "Duration for the Performance test to run, Default:30s")
```